tests/unit/lib/package/test_s3_uploader.py

```diff
@@ -52,6 +53,18 @@ def test_s3_uploader_artifact_metadata(self):
         with self.assertRaises(TypeError):
             s3_uploader.artifact_metadata = "Not a dict"
 
+    def test_s3_uploader_artifact_acl(self):
+        s3_uploader = S3Uploader(
+            s3_client=self.s3,
+            bucket_name=self.bucket_name,
+            prefix=self.prefix,
+            kms_key_id=self.kms_key_id,
+            force_upload=self.force_upload,
+            no_progressbar=self.no_progressbar,
+        )
+        s3_uploader.artifact_acl = "bucket-owner-full-control"
+        self.assertEqual(s3_uploader.artifact_acl, "bucket-owner-full-control")
+
```

Comment:
I believe we also need a test case to test `s3_uploader.upload` with `acl` specified

Reply:
@celestialorb Can we address this please? Thanks!